.chloggen/add_shutdown_podman.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: bug_fix
```
Nit: I think bug fix may be slightly more accurate.